source/manual/get-started.html.md

```diff
@@ -17,24 +17,296 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
+1. Ask somebody with access to add your GitHub username to the [user monitoring system][user-reviewer].
+1. [Generate and register an SSH key pair][register-ssh-key] for your laptop for your GitHub account. You should use a `4096` bit key.
+1. Import the SSH key into your keychain. Once you’ve done this, you'll be able to clone repos over SSH.
+
+    ```
+    $ /usr/bin/ssh-add -K your-private-key
+    ```
+
+1. Add the above line into your `~/.bash_profile` or equivalent so that it is persistent between restarts.
+    
+1. Test that it all works by running `ssh -T git@github.com`.
+
+1. While you're here, associate your name and email to your git commits:
+
+    ```
+    $ git config --global user.email "friendly.giraffe@digital.cabinet-office.gov.uk"
+    $ git config --global user.name "Friendly Giraffe"
+    ```
+
+[alphagov]: https://github.com/alphagov
+[GitHub]: https://www.github.com/
+[govuk-team]: https://github.com/orgs/alphagov/teams/gov-uk/members
+[register-ssh-key]: https://help.github.com/articles/connecting-to-github-with-ssh/
+[user-reviewer]: https://github.com/alphagov/govuk-user-reviewer
+
+## 3. Install GDS tooling
+
+On GOV.UK we use two command-line tools day-to-day: [`govuk-connect`](https://github.com/alphagov/govuk-connect) and the [`gds-cli`](https://github.com/alphagov/gds-cli) for AWS, SSH and VPN access.
+
+To install these, run:
+
+```bash
+brew tap alphagov/gds
+brew install gds-cli govuk-connect
+```
+
+The GDS CLI repository is private, so you'll need to follow the GitHub setup instructions above for the download to work.
+
+Test that both tools work by running `gds --help` and `gds govuk connect --help`.
+
+> If you see `fatal: no such path in the working tree`, it's because you're using ZSH, which has `gds` set up as a Git alias. You can either remove that alias by adding `unalias gds` to your `~/.zshrc`, or use `gds-cli` instead of `gds` for all the relevant commands.
+
+The GDS CLI requires some initial configuration:
+
+```bash
+gds config email firstname.lastname@digital.cabinet-office.gov.uk
+gds config yubikey false # If you type MFA codes from your phone
+```
+
+## 4. Connecting to the GDS VPN
+
+Access to our infrastructure and internal services is controlled by IP safelisting. If you're outside of the office or not on the Brattain network (ie, you're on GovWiFi), you'll need to connect to the GDS VPN to access our stuff.
+
+To do this, read [the GDS Wiki page about the VPN][gds-vpn-wiki] to ensure you have the required pre-requisites, for example the Cisco AnyConnect software installed on your computer, and an MFA token given to you when you arrived by GDS IT. If you don't have these, [contact the IT helpdesk][gds-it-helpdesk].
+
+The `gds-cli` tool that we installed earlier has a `vpn` subcommand.
+
+To connect to the VPN, run `gds vpn connect` and you'll be asked for your LDAP password and MFA code. To disconnect, run `gds vpn disconnect`.
```

Comment:
What does LDAP stand for?

Reply:
LDAP is the protocol that we authenticate to GDS IT services with (ie when we login to our GDS laptops, print things, etc.) I wanted to say "laptop password", but for people on their own devices that's incorrect. I've asked IT what better words there are for this. Maybe "GDS IT password"? Will people understand that?

Reply:
```suggestion
To connect to the VPN, run `gds vpn connect` and you'll be asked for a password and MFA code. The password is the one you use to login to your GDS-issued laptop or the printers.

To disconnect, run `gds vpn disconnect`.
```

Reply:
I think GDS IT password is ok, I like your suggested change a lot though! Doesn't leave much room for doubt.